src/configuration/IM/ejabberd/14.12/ejabberd.yml

```diff
@@ -0,0 +1,694 @@
+      # always at the top of their list, we enforce the server cipher order
+      # using the setting "cipher_server_preference"
+      # The cipher order is defined using the "ciphers" option (see below)
+      - "cipher_server_preference"
+    max_stanza_size: 65536
+    # Optionally use your own Diffie-Hellman-Parameters
+    # Generate the file using "openssl dhparam -out dhparams.pem 4096"
+    dhfile: "/path/to/dhparams.pem"  # Available from version 15.06
+    shaper: c2s_shaper
+    access: c2s
+    # Set the cipher order of the server, in OpenSSL syntax
+    ciphers: "EDH+CAMELLIA:EDH+aRSA:EECDH+aRSA+AESGCM:EECDH+aRSA+SHA256:EECDH:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!DSS:!RC4:!SEED:!IDEA:!ECDSA:kEDH:CAMELLIA128-SHA:AES128-SHA"
+    # TODO Is this cipher order fine?
+  - 
+    port: 5269
+    module: ejabberd_s2s_in
+  ##
+  ## ejabberd_service: Interact with external components (transports, ...)
+  ##
+  ## - 
+  ##   port: 8888
+  ##   module: ejabberd_service
+  ##   access: all
+  ##   shaper_rule: fast
+  ##   ip: "127.0.0.1"
+  ##   hosts:
+  ##     "icq.example.org":
+  ##       password: "secret"
+  ##     "sms.example.org":
+  ##       password: "secret"
+
+  ##
+  ## ejabberd_stun: Handles STUN Binding requests
+  ##
+  ## - 
+  ##   port: 3478
+  ##   transport: udp
+  ##   module: ejabberd_stun
+
+  ##
+  ## To handle XML-RPC requests that provide admin credentials:
+  ##
+  ## - 
+  ##   port: 4560
+  ##   module: ejabberd_xmlrpc
+  - 
+    port: 5280
+    module: ejabberd_http
+    ## request_handlers:
+    ##   "/pub/archive": mod_http_fileserver
+    web_admin: true
+    http_poll: true
+    http_bind: true
+    ## register: true
+    captcha: true
+    # Enable SSL for web admin
+    certfile: "/path/to/ssl.pem"
+    tls: true
+
+##
+## s2s_use_starttls: Enable STARTTLS + Dialback for S2S connections.
+## Allowed values are: false optional required required_trusted
+## You must specify a certificate file.
+##
+s2s_use_starttls: required_trusted
+# This forbids S2S-connections to servers with selfsigned certs. If you need
+# that capability, change this setting to "required" to require encryption,
+# but not authentication
+
+##
+## s2s_certfile: Specify a certificate file.
+##
+s2s_certfile: "/path/to/ssl.pem"
+
+## Custom DH parameters
+# Generate using "openssl dhparam -out dhparams.pem 4096"
+s2s_dhfile: "/etc/ejabberd/dhparams.pem"  # Available from version 15.03
+## Custom OpenSSL options
+##
+s2s_protocol_options:
+  - "no_sslv3"
+  - "no_tlsv1"
+  - "cipher_server_preference"
+  # Again, prefer our known good cipher ordering to the potentially bad one
+  # of the other server
+
+s2s_ciphers: "EDH+CAMELLIA:EDH+aRSA:EECDH+aRSA+AESGCM:EECDH+aRSA+SHA256:EECDH:+CAMELLIA128:+AES128:+SSLv3:!aNULL:!eNULL:!LOW:!3DES:!MD5:!EXP:!PSK:!DSS:!RC4:!SEED:!IDEA:!ECDSA:kEDH:CAMELLIA128-SHA:AES128-SHA"
+# TODO Is this cipher order fine?
```

Comment:
TODO can be removed, as the string is the same as in `src/common/cipherStringB.tex`.